src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1051,7 +1051,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // MSFT 33353327: We're purposefully not using _initializedTerminal to ensure we're fully initialized.
         // Doing so makes us return nullptr when XAML requests an automation peer.
         // Instead, we need to give XAML an automation peer, then fix it later.
-        if (!_IsClosing())
+        if (!_IsClosing() && !_detached)

```

Comment:
Does it make sense to instead check for `!_interactivity` since it's used below?

Reply:
When termcontrol isn't detached or closing we expect `_interactivity` to be non-null, no?

Given #10971 has happened in the past, I think `THROW_HR_IF_NULL(E_UNEXPECTED, _interactivity)` is better suited.

Reply:
I'm OK with this for now